pkg/proto/message.go

```diff
@@ -564,3 +569,42 @@ func (rq *ListResponse) Unmarshal(b []byte) error {
 	}
 	return nil
 }
+
+// CreateTopicRequest
+//-------------------------
+
+// Marshal ...
+func (rq CreateTopicRequest) Marshal() ([]byte, error) {
+	buf := bytes.NewBuffer(binary.BigEndian.AppendUint32([]byte{}, uint32(len(rq.Topic))))
+	_, err := buf.Write([]byte(rq.Topic))
+	if err != nil {
+		return nil, err
+	}
+	_, err = buf.Write([]byte(rq.Schema))
+	if err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+// Unmarshal ...
+func (rq *CreateTopicRequest) Unmarshal(b []byte) error {
+	buf := bytes.NewBuffer(b)
+	lengthPrefix := make([]byte, lenWidth)
+	n, err := io.ReadFull(buf, lengthPrefix)
+	if err != nil {
+		return err
+	}
+	length := binary.BigEndian.Uint32(lengthPrefix)
+	topic := make([]byte, length)
+	m, err := io.ReadFull(buf, topic)
+	if err != nil {
+		return err
+	}
+	rq.Topic = string(topic)
+	rq.Schema = string(b[n+m:])
```

Comment:
For some reason, I feel like this might error for empty strings.

Reply:
This appears to work, added test coverage to ensure it does though.